TestProjects/ShaderGraph/SaveLoadGraphs/SimpleWithSubGraph.shadergraph

```diff
@@ -1124,7 +1124,7 @@
     },
     "m_Name": "A",
     "m_DefaultReferenceName": "Vector1_e29cd19dafd54a1dbd31dcf1d75c21b2",
-    "m_OverrideReferenceName": "a_3_4_d__",
```

Comment:
Fixing an issue with this test on GL devices -- not allowed to use double underscore in identifiers.